prime-angular-frontend/src/app/modules/paper-enrolment/pages/care-setting-page/care-setting-page.component.ts

```diff
@@ -55,6 +58,15 @@ export class CareSettingPageComponent extends AbstractEnrolmentPage implements O
     this.routeUtils.routeRelativeTo([PaperEnrolmentRoutes.DEMOGRAPHIC]);
   }
 
+  public disableCareSetting(careSettingCode: number): boolean {
+    switch (careSettingCode) {
```

Comment:
Switch cases are used when you have a lot of possible results, but in this case you have 2.  This should be:

```ts
return (careSettingCode === CareSettingEnum.DEVICE_PROVIDER)
  ? !this.permissionService.hasRoles(Role.FEATURE_SITE_DEVICE_PROVIDER);
  : false
```

Reply:
Updated